components/schemas/monitoring/logs/LogsSearch.yml

```diff
@@ -0,0 +1,14 @@
+title: LogsSearch
+required:
+  - type
+  - match
+properties:
+  type:
+    description: The input type used to query logs.
```

Comment:
can you expand on what raw/regex mean in context of querying logs? "When set to regex, ...."